### PR TITLE
Send less initial pings for healthcheck

### PR DESCRIPTION
**Pull request by LefterisJP**
Previously when we first create a queue to a node we would end up sending too many pings to it to find out if it's healthy and in the meantime consider it as reachable.

We would send 120 pings, waiting 60 seconds between each ping for the
initial healthcheck. In the meantime, raiden would consider the node
healthy and try to route transfers through it.

Now for that special initial healthcheck we make sure to use a lot less
pings and immediately consider it unhealthy if this does not work.

### CODE REVIEW

File: raiden/network/protocol.py [hunk 1]
```diff
@@ -327,7 +329,7 @@ def healthcheck(
             data,
             receiver_address,
             event_stop,
-            [nat_keepalive_timeout] * nat_keepalive_retries,
+            [DEFAULT_INITIAL_HEALTHCHECK_TIMEOUT] * DEFAULT_INITIAL_HEALTHCHECK_RETRIES,
```
Comment (hackaugusto) [inline raiden/network/protocol.py]: There is no need for the new configuration variables, `nat_keepalive_timeout` and `nat_keepalive_retries` are set through the configuration variables `DEFAULT_NAT_KEEPALIVE_TIMEOUT` and `DEFAULT_NAT_KEEPALIVE_RETRIES` and are properly propagated from the ui to the healthcheck.

File: raiden/settings.py [hunk 2]
```diff
@@ -25,3 +25,6 @@
 DEFAULT_NAT_KEEPALIVE_RETRIES = 5
 DEFAULT_NAT_KEEPALIVE_TIMEOUT = 30
 DEFAULT_NAT_INVITATION_TIMEOUT = 180
+
+DEFAULT_INITIAL_HEALTHCHECK_TIMEOUT = 5
```
Comment (hackaugusto) [inline raiden/settings.py]: Please, change `--max-unresponsive-time` at the raiden ui to 25 to get the same effect.

Comment (hackaugusto) [reply raiden/settings.py]: Humm, actually `--send-ping-time` needs fixing, it's not using the default value

Comment (LefterisJP) [reply raiden/settings.py]: With --max-unresponive time to `25` it's still quite a long waiting time ([60, 60, 60, 60, 60])
and I argue that the initial ping should be a lot more strict than the later ones.

**Pull request merged**